[Build] Buildfailure when enabling 'CMAKE_BUILD_TYPE=Release'

## Issue Description
If `CMAKE_BUILD_TYPE=Release` is defined when running cmake command, `unused variable` occurs as below. The main reason is that **NDEBUG** is automatically defined when building `Release` mode

## How to Reproduce

```bash
$ mkdir build & cd build
$ cmake .. -DCMAKE_BUILD_TYPE=Release
$ make
```

## Error message

```bash
[ 69%] Built target nnstreamer_customfilter_scaler_allocator
/home/again4you/nnstreamer_work/ref_repo/nnstreamer/nnstreamer_example/custom_example_average/nnstreamer_customfilter_example_average.c: In function ‘set_inputDim’:
/home/again4you/nnstreamer_work/ref_repo/nnstreamer/nnstreamer_example/custom_example_average/nnstreamer_customfilter_example_average.c:61:12: error: unused variable ‘data’ [-Werror=unused-variable]
   pt_data *data = private_data;
            ^
[ 72%] Built target nnstreamer_customfilter_opencv_scaler
/home/again4you/nnstreamer_work/ref_repo/nnstreamer/nnstreamer_example/custom_example_average/nnstreamer_customfilter_example_average.c: In function ‘pt_invoke’:
/home/again4you/nnstreamer_work/ref_repo/nnstreamer/nnstreamer_example/custom_example_average/nnstreamer_customfilter_example_average.c:112:12: error: unused variable ‘data’ [-Werror=unused-variable]
   pt_data *data = private_data;
            ^
```

--- a/nnstreamer_example/custom_example_average/nnstreamer_customfilter_example_average.c
+++ b/nnstreamer_example/custom_example_average/nnstreamer_customfilter_example_average.c
@@ -58,9 +58,8 @@ set_inputDim (void *private_data, const GstTensorFilterProperties * prop,
     const GstTensorsInfo * in_info, GstTensorsInfo * out_info)
 {
   int i;
-  pt_data *data = private_data;
 
-  assert (data);
+  assert (private_data);
   assert (in_info);
   assert (out_info);
 
@@ -109,7 +108,6 @@ static int
 pt_invoke (void *private_data, const GstTensorFilterProperties * prop,
     const GstTensorMemory * input, GstTensorMemory * output)
 {
-  pt_data *data = private_data;
   uint32_t c, x, y, z;
 
   uint32_t ix = prop->input_meta.info[0].dimension[0];
@@ -124,7 +122,7 @@ pt_invoke (void *private_data, const GstTensorFilterProperties * prop,
       prop->input_meta.info[0].dimension[1] *
       prop->input_meta.info[0].dimension[2];
 
-  assert (data);
+  assert (private_data);
   assert (input);
   assert (output);
 

